### DISCUSSION TITLE
Missing bundle registration/arp entry for all permanent chain

### DISCUSSION THREAD

**Discussion by ds-andreasor** [wixtoolset/issues#8032] [Questions] [answered]
Something I found while migrating a bundle from v3 to v4 (v5).
If all of the chain packages have `Permanent="yes"` the bundle itself does not show up in the registry and under add remove programs.

I searched the documentation but did not find a behaviour change that would explain this.

Is this the intended behaviour?

I am not sure if it really makes sense to have a bundle registration with packages that are all independant of the bundle after the install.
Of course such a bundle does never detect itself as installed.

**Comment by barnson** [comment, marked as answer]
https://github.com/wixtoolset/burn/pull/35 has the logic.
